config/default_records/asset_shapes/default_records.yml

```diff
@@ -0,0 +1,62 @@
+# This file contains the AssetShape and associated Map records to be created
+# in deployment environments by RecordLoader. The loading is triggered by
+# the "post_deploy" task. The records are checked against the database by
+# their names. If the records already exist, they are not created.
+# AssetShape records are created in the "asset_shapes" table, and Map records
+# are created in the "maps" table. Each section in this file starts with the
+# name of the AssetShape record. If the name is defined again in the options,
+# it takes priority. The "horizontal_ratio" and "vertical_ratio" options
```

Comment:
I didn't know what you meant by "If the name is defined again in the options, it takes priority." 
But I think I figured it out, you mean if there's `name: A different name`, for instance? Might be worth saying that in case it's not clear to anyone else.

Reply:
I have removed the comment and updated the loader as well because it sounds like this is the behaviour of all record loaders, which is misleading. I will push change shortly.